crates/typst/src/eval/methods.rs

```diff
@@ -16,6 +16,7 @@ pub fn is_accessor(method: &str) -> bool {
 
 /// List the available methods for a type and whether they take arguments.
 pub fn mutable_methods_on(ty: Type) -> &'static [(&'static str, bool)] {
+    // TODO: does Version::at need to be listed here?
```

Comment:
Only if we want `version.at(2) = 3` to work. I think it's fine if versions are immutable to be honest.